.github/workflows/close_pr.yml

```diff
@@ -0,0 +1,62 @@
+name: Cleanup when PR closed
+
+on:
+  pull_request:
+    types: [ closed ]
+
+jobs:
+  cleanup:
+    name: Clean up Azure
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add comment for start of job
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Stopping reviewapp and clearing up
+
+      - name: Log in to Azure
+        uses: azure/login@v1
+        with:
+          creds: '{"clientId":"${{ vars.ARM_CLIENT_ID }}","clientSecret":"${{ secrets.ARM_CLIENT_SECRET }}","subscriptionId":"${{ vars.ARM_SUBSCRIPTION_ID }}","tenantId":"${{ vars.ARM_TENANT_ID }}"}'
+
+      - name: Remove label
+        id: remove_label
+        env:
+          ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
+          REF_NAME: ${{ github.head_ref }}
+        run: |
+          ENV_SLUG=$(echo "${REF_NAME}" | sed -e 's:[^[:alpha:]|^[:digit:]]:-:g' | sed -e 's/\(.*\)/\L\1/')
+          az containerapp revision label remove --label ${ENV_SLUG} --name portal -g rg-ecc-portal-uks-dev --subscription "Essex County Council (Portal)"
+          sleep 1m
```

Comment:
This sleep is here because I found that the next job started too quickly and didn't deactivate the revision.